src/util/irange.h

```diff
@@ -0,0 +1,90 @@
+// Copyright (c) 2021 The Dash Core developers
```

Comment:
along with referring original implementation as a source, adding their copyright is probably for the best, also mention the commit of cpp_range from which this was derived from

Reply:
They didn't have a copyright on the file, just a MIT license in the folder